utils/getFeed.js

```diff
@@ -27,6 +27,13 @@
  * @returns {string|undefined} The string result of the xpath query.
  */
 function getXPathText( xpath, document, context = null, nsResolver = null ) {
```

Comment:
This function was specifically meant to get the posts feed for Yoast.com. I would like to see these functions to be refactored into specific functions for parsing a posts and a courses feed.